extensions/cleanshotx/src/add-quick-access-overlay.tsx

```diff
@@ -0,0 +1,10 @@
+import { LaunchProps, closeMainWindow, open } from "@raycast/api";
+
+export default async function Command(props: LaunchProps<{ arguments: Arguments.AddQuickAccessOverlay }>) {
+  let url = "cleanshot://add-quick-access-overlay";
+
+  url += `?filepath=${encodeURIComponent(props.arguments.filepath)}`;
+
```

Comment:
@erics118

Reply:
> Do you think we should either open the path on clipboard and or open the file picker view (finder).
> 
> Instead of manually typing the path for the specific file we want to open every single time?

The other commands have the filepath parameter as optional, as CleanShot X opens the file picker for you. But, for this one, filepath is a required argument, so I maintained it to be the same way, where you need to input the file path

Reply:
If you say so! LGTM then.